application/src/main/java/net/furizon/backend/feature/pretix/event/Event.java

```diff
@@ -0,0 +1,50 @@
+package net.furizon.backend.feature.pretix.event;
+
+import lombok.Builder;
+import lombok.Data;
+import lombok.RequiredArgsConstructor;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.springframework.data.util.Pair;
+
+import java.time.OffsetDateTime;
+import java.util.Set;
+
+@RequiredArgsConstructor
+@Data
+@Builder
+public class Event {
+    @NotNull
+    private final String slug;
+
+    @Nullable
+    private OffsetDateTime dateTo;
+
+    @Nullable
+    private OffsetDateTime dateFrom;
+
+    private boolean isCurrent;
+
+    @NotNull
+    private String publicUrl;
+
+    @Nullable
+    private Set<String> eventNames;
+
+    public static class EventBuilder {
```

Comment:
EventBuilder not building Event class

Reply:
Here the EventBuilder static class is used just to extend the lombok's @Builder annotation with a new method